3.evaluate_model/scripts/nbconverted/apply_tsne.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Apply tsne to each feature dataset
+# 
+# Input: Data representations
+# Output: tsne embeddings for plotting
+
+# In[1]:
+
+
+import pathlib
+import numpy as np
+import pandas as pd
+from sklearn.manifold import TSNE
+
+import sys
+
+sys.path.append("../utils")
+from split_utils import get_features_data
+
+
+# In[2]:
+
+
+np.random.seed(1234)
+
+
+# In[3]:
+
+
+output_file = pathlib.Path("evaluations", "tsne_embeddings.csv.gz")
```

Comment:
Is there a specific reason to use compressed CSV over a parquet file?

Reply:
habit! IMO it's ok to keep compressed csv